docs/patternlib/tutorials/workshop.md

```diff
@@ -81,7 +82,7 @@ jazz voodoo birds3 procshort blip drum jvbass psr wobble drumtraks koy
 rave bottle kurt latibro rm sax lighter lt arpy feel less stab ul
 ```
 
-You can see what other sounds there are in the [default library](/configuration/AudioSamples/default_library.md) by looking in the `Dirt-Samples` folder. Find it via the `SuperCollider` menu: `'File > Open user support directory > downloaded-quarks > Dirt-Samples'`. Additionally, you can also add your own [custom samples](/configuration/AudioSamples/samples.md).
+You can see what other sounds there are in the [default library](/configuration/AudioSamples/default_library.md) by looking in the `Dirt-Samples` folder. Find it via the `SuperCollider` menu: `'File > Open user support directory > downloaded-quarks > Dirt-Samples'`. Additionally, you can also add your own [custom samples](/configuration/AudioSamples/samples.md). In the Pulsar editor, you can add a setting that will load a tab with all the Dirt-Samples. (see [Pulsar](/getting-started/editor/Pulsar))
```

Comment:
I think is best: "In the Pulsar editor, you can add a setting that will load a tab with all the Dirt-Samples (see [Pulsar](/getting-started/editor/Pulsar.md)).", with the "." at the end, and the file extension ".md" in the link.

Reply:
Yes. Thanks for the correction! I've added a commit with this change. Should be ready to merge.